src/model/configuration.go

```diff
@@ -67,19 +67,26 @@ type Configuration struct {
 	SunAngle              int       `json:"sun_angle"`
 	Track                 string    `json:"track"`
 	TrackConfig           string    `json:"track_config"`
+	LegalTyres            string    `json:"legal_tyres"`
+	UdpPluginPort         int       `json:"udp_plugin_local_port"`
+	UdpPluginAddr         string    `json:"udp_plugin_address"`
```

Comment:
Is UdpPluginAddr supposed to be a string? You used a checkbox in the frontend and another checkbox for UdpPluginPort, which is an int. I think you need to change them to number fields in the UI (and add parseInt() if you haven't done that yet).

At the moment I get a marshalling error: `json: cannot unmarshal bool into Go struct field Configuration.udp_plugin_address of type string`